src/browser/clusterTask.spec.ts

```
@@ -426,4 +422,176 @@ describe('generatePdf', () => {
       expect(global.fetch).toHaveBeenCalledTimes(2);
     });
   });
+
+  describe('response handler', () => {
+    function getResponseHandler(): (response: unknown) => Promise<void> {
+      const calls = mockPage.on.mock.calls.filter(
+        ([event]: [string]) => event === 'response',
+      );
+      return calls[calls.length - 1][1] as (
+        response: unknown,
+      ) => Promise<void>;
```

ESLint

[error] Replace `⏎········response:·unknown,⏎······` with `response:·unknown`

(prettier/prettier)